alerts/confluence_shell.py

```diff
@@ -1,46 +0,0 @@
-#!/usr/bin/env python
```

Comment:
This is deprecated as this system is no longer on our network. thus the removal.